quil-rs/src/program/graph.rs

```diff
@@ -329,11 +329,13 @@ impl<'a> InstructionBlock<'a> {
                     Ok(())
                 }
                 InstructionRole::RFControl => {
-                    let matched_frames = program.get_frames_for_instruction(instruction);
+                    let matched_frames: Option<MatchedFrames<'_>> =
```

Comment:
Does the custom handler make the explicit type here necessary, or did you add it for some other reason?